### PR TITLE
0.11.0 version bump

### PR DESCRIPTION
**Pull request by pawelchcki**
### Description

Update version to 0.11.0  + add verification steps to ensure consistency

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -97,4 +91,19 @@ $(PACKAGES_BUILD_DIR):
 packages: .apk .rpm .deb .tar.gz
 	tar -zcf packages.tar.gz $(PACKAGES_BUILD_DIR)
 
-.PHONY: dist_clean clean all install sudo_install test_c test_c_mem test test_integration install_ini install_all .apk .rpm .deb .tar.gz src/ext/version.h sudo debug run-tests.php
+verify_pecl_file_definitions:
```
Comment (SammyK) [inline Makefile]: ❤️

File: package.xml [hunk 2]
```diff
@@ -12,8 +12,8 @@
     </lead>
     <date>2019-01-21</date>
     <version>
-        <release>0.10.0</release>
-        <api>0.10.0</api>
+        <release>0.11.0</release>
+        <api>0.11.0</api>
```
Comment (SammyK) [inline package.xml]: We just need to update `<notes>` with the CHANGELOG stuff as well. :)

File: package.xml [hunk 3]
```diff
@@ -12,8 +12,8 @@
     </lead>
     <date>2019-01-21</date>
```
Comment (SammyK) [inline package.xml]: Just a little time warp. :)
```suggestion
    <date>2019-01-23</date>
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,11 +3,13 @@ All notable changes to this project will be documented in this file - [read more
 
 ## [UNRELEASED]
 
+## [0.11.0]
```
[the remark this answers is not in the document]
Comment (pawelchcki) [reply CHANGELOG.md]: Doh... 🤦‍♂️

File: package.xml (hunk 2 above)
Comment (pawelchcki) [reply package.xml]: Ohhhh 🤦‍♂️

**Comment by pawelchcki** [comment]
Got to 🤕 from all the 🤦‍♂️ about forgotten steps - thats why I like automation :D

**Pull request merged**